lib/jose-utils.js

```diff
@@ -367,5 +367,15 @@ Utils.sha256 = function(str) {
 Utils.isCryptoKey = function(rsa_key) {
   // Some browsers don't expose the CryptoKey as an object, so we need to check
   // the constructor's name.
-  return rsa_key.constructor.name == 'CryptoKey';
+  if (rsa_key.constructor.name == 'CryptoKey') {
```

Comment:
I figured that might be an issue. I wonder if this library should have some kind of flag to enable non-native browser crypto. It might be overkill, I'm just trying to figure out how to prevent a random library user who doesn't know much about crypto to end up accidentally weakening things.